airbyte-integrations/connectors/source-airtable/integration_tests/expected_records.txt

```diff
@@ -0,0 +1,42 @@
+{"stream": "users/table_1", "data": {"_airtable_id": "recgRMRMHxgcgJeDe", "_airtable_created_time": "2021-11-16T13:30:17.000Z", "name": "test2", "status": "test", "notes": "test_note2"}, "emitted_at": 1674657815126}
```

Comment:
please use the `.jsonl` extension instead of `.txt`

Reply:
Tricky part) It used to be `txt`  as far as I remember [here](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference/#example-of-expected_recordsjsonl). 

Sure, thanks.